src/components/learning-units-list/LearningUnitListItem.js

```diff
@@ -1,44 +1,44 @@
-import useGet from '@hooks/useGet';
-import Image from 'next/image';
-import Link from 'next/link';
-import { Button } from 'primereact/button';
-import { ToggleButton } from 'primereact/togglebutton';
 import { Card } from 'primereact/card';
+import Image from 'next/image';
 import styles from '@styles/LearningUnitsList.module.scss';
 import profilePic from '@utils/images/unit.jpeg';
+import LinkButton from './LinkButton';
+import { CompleteLearningUnitToggle } from './CompleteLearningUnitToogle';
 import { endpoints } from '@utils/endpoints';
+import useGet from '@hooks/useGet';
+import { Skeleton } from 'primereact/skeleton';
 
 function LearningUnitItem({ unit }) {
-  const completedLearningUnit = endpoints('isLearningUnitCompleted', unit.id);
+  const completedLearningUnitEndpoint = endpoints('isLearningUnitCompleted', unit.id);
 
-  const { data: isCompleted, isLoading, isError, mutate } = useGet(completedLearningUnit);
+  const { data: isCompleted, isLoading, isError, mutate } = useGet(completedLearningUnitEndpoint);
 
-  const handleOnChange = (clicked) => {
+  const changeHandler = (clicked) => {
     if (clicked.value) {
       const requestOptions = {
         method: 'POST',
         headers: { 'Content-Type': 'application/json' },
       };
-      fetch(completedLearningUnit, requestOptions).then((response) => {
+      fetch(completedLearningUnitEndpoint, requestOptions).then((response) => {
         if (response.ok) {
-          mutate(completedLearningUnit);
+          mutate(completedLearningUnitEndpoint);
```

Comment:
mutate is already defined in this file -> doesn't require parameters

Reply:
Amo mutate (ok!).